apps/cowswap-frontend/src/modules/trade/pure/TradeConfirmation/styled.tsx

```diff
@@ -58,6 +58,10 @@ export const ContentWrapper = styled.div`
   gap: 10px;
   font-size: 14px;
   padding: 0 10px 10px;
+
+  .modalMode & {
+    /* padding: 0; */
+  }
```

Comment:
Did you mean to leave this commented out?

Reply:
Oops will clean up